package.json

```diff
@@ -49,7 +49,7 @@
     ]
   },
   "main": "src/index.js",
-  "name": "excalidraw",
+  "name": "allow-dragging-transparent-shapes-when-selected",
```

Comment:
First time making a PR from codesandbox.io - I assume this needs to be reverted :P probably automatically happened when I changed the sandbox title...